frontend/src/contexts/AuthContext.tsx

```diff
@@ -61,7 +61,13 @@ const AuthContextProvider: FC<PropsWithChildren<{}>> = props => {
   }, []);
 
   return (
-    <AuthContext.Provider value={{ isLoggedIn, token, userId }}>
+    <AuthContext.Provider
+      value={{
+        isLoggedIn,
+        token,
+        userId,
+      }}
+    >
```

Comment:
I can't see any actual modification here 🧐

Reply:
I created another context for the Tenant, because the Tenant have to wait  for the authContext login successfully